lib/scope.js

```diff
@@ -605,72 +592,26 @@ module.exports = {
       // button, radio, checkbox yesno, and maybe others need the value of the `value` attribute
       // Anything that has a `value` attribute, but has not yet calculated its own value
       if ( values.length === 0 && node.attribs.value ) {
-        // TODO: Add docs about da YAML values that devs never fill in themselves and
-        // so don't know what they are. E.g. yesnomabye -> True/False/None
+        // TODO: Add links to docs about da YAML values that devs never fill in themselves and
+        // so don't know what they are. E.g. yesnomabye -> True/False/None.
         // TODO: Text inputs and textareas must ignore the value, etc. How?
         values.push( node.attribs.value );
       }
-      // TODO: add default of empty string for values?
 
-      field_data.var_names = var_names;
-      field_data.values = values || [];
+      // ******************
 
-      // We need to try the full names of each field. If there are proxy vars (x, i, etc)
-      // this will let us find which one matches the sought_var and we can use it to
-      // create a map from proxies to intrinsic names. Ex: {x: "foo"}.
-      // We will then need to use the data gathered here to convert each field to
-      // have its intrinsic name.
-      field_data.name_patterns = [];
+      // To allow more felxibility for devs of simpler interviews,
+      // will not require sought var unless it's necessary.
       for ( let name of var_names ) {
-        // Get the data that will help us construct the intrinsic name later and
-        // look for the sought_var match now.
-        let pattern = await scope.getSoughtVarPatterns( scope, { field_name: name, });
-        field_data.name_patterns.push( pattern );
-        // Try to pull out the intrinsic name parts
-        let maybe_map = await scope.getProxiesToNamesMap( scope, {
-          sought_var: page_data.sought_var, pattern_data: pattern,
-        });
-        // If that works, it's the right map for the proxy values
-        if ( Object.keys( maybe_map ).length ) {
-          proxies_to_names_map = maybe_map;
+        if ( proxies_regex.test( name )) {
```

Comment:
Will find `x`, `i`, `j`, etc - the proxy vars. If any field has a proxy var, the whole page data will store that.

Reply:
From @niharikasingh 
> Do we need to add tests to make sure all appearances of proxy vars are caught?

From me:
I think we can. I think we may not need to because the regex finds an isolated letter of the type described, so it's pretty simple. It can fail if someone uses `foo.i` and we can make that more robust, but that's bad practice. I think they can't use `x` for other purposes in da because `x` is special. The also can't use `[i]`, `[j]`, etc because those are special. This is very stringent and I think it's one thing we really can count on.

Reply:
Add to proxy var regex - check for brackets.
^ to make an issue with.

Reply:
Comment added. #261 made for brackets regex.